kits/default/default.info.hidden.yml

```diff
@@ -2,7 +2,7 @@ name: OE_BOOTSTRAP_THEME_SUBTHEME_NAME
 description: OE_BOOTSTRAP_THEME_SUBTHEME_DESCRIPTION
 screenshot: screenshot.png
 type: theme
-base theme: oe_bootstrap_theme
+base theme: OE_BOOTSTRAP_THEME_SUBTHEME_MACHINE_NAME
 core_version_requirement: ^8.9 || ^9.1
 
 stylesheets-remove:
```

Comment:
stylesheets-remove is deprecated in D8 and removed in D9, we should use libraries-override